README.md

```diff
@@ -6,49 +6,50 @@ Docker development environment for RAS
 
 ##  Clone and run the rm-docker-dev environment
  
-```
-   git clone  https://github.com/jwgwalton/rm-docker-dev.git (FIXME: MOVE TO ONSSDIGITAL WHEN MERGED)
+```bash
+   git clone  https://github.com/jwgwalton/rm-docker-dev.git # (FIXME: MOVE TO ONSSDIGITAL WHEN MERGED)
```

Comment:
The changes have gone into onsdigital rm-docker-dev so can remove the fix and change url to git clone  https://github.com/ONSdigital/rm-docker-dev.git.